doc/release-notes.md

```diff
@@ -33,6 +33,25 @@ frequently tested on them.
 Notable changes
 ===============
 
+Performance Improvements
+--------------
+
+Validation speed and network propogation performance have been greatly
+improved, leading to much shorter sync and initial block download times.
+
+- The script signature cache has been reimplemented as a high-performance,
+  low-overhead "cuckoo cache"
```

Comment:
...", allowing for more signatures to be cached and cache lookups to be performed much faster."